### ISSUE TITLE
Build Instructions

### ISSUE THREAD

**Issue by jasonwilliams** [albertziegenhagel/childdebugger-vscode#8]
Hey @albertziegenhagel 

Do you have instructions on how you've been building this extension? I would love to play with it but don't know what steps you took (sorry my C/C++ foo is rough)

Thanks

**Comment by albertziegenhagel** [comment]
@jasonwilliams Could you retry your application with the newest pre-release? Note that the new `*.vsix` package from https://github.com/albertziegenhagel/childdebugger-vscode/releases/tag/head includes the debug engine extension binaries and automatically integrates with the debug engine, so I would suggest you delete the old, manual integration - especially the `*.link` file in `$HOME/.cppvsdbg/extensions` - and then just install the new extension package in VS Code.

I fixed a few issues that could have been the reason for your issues. Mainly:

  - 067fe9c5827f874f92bf001c70fdcea2c5ea9e65, which fixed an issue that could lead to the debugger crashing
  - #17 which fixed an issue that could lead to the debugger not attaching correctly to children of children

**Comment by jasonwilliams** [comment]
Yes I believe part of it is a 32bit application (what makes you ask this?). I will try this and get back to you, thanks!

**Comment by albertziegenhagel** [comment]
Good to hear that you are not seeing any of the errors anymore. Let me know if there is anything else I can help with.
